Apps/GatewayApi/test/unit/Services.Test/UserProfileServiceTests.cs

```diff
@@ -344,7 +344,7 @@ public async Task ShouldQueueNotificationUpdate()
             // Act
             RequestResult<UserProfileModel> actualResult = await service.CreateUserProfile(
                     new CreateUserRequest
-                        { Profile = userProfile },
```

Comment:
This does not match our formatting template. I think this should be discussed with team... @MikeLyttle